stripe/deploy.gradle

```diff
@@ -58,80 +50,95 @@ afterEvaluate { project ->
                 artifact androidJavadocsJar
                 artifact androidSourcesJar
 
-                // You can then customize attributes of the publication as shown below.
-                groupId = GROUP
-                artifactId = POM_ARTIFACT_ID
+                groupId = "com.stripe"
+                artifactId = "stripe-android"
                 version = VERSION_NAME
+
+                pom {
+                    name = "stripe-android"
+                    packaging = "aar"
+                    description = "Stripe Android SDK"
+                    url = "https://github.com/stripe/stripe-android"
+
+                    scm {
+                        url = "https://github.com/stripe/stripe-android"
+                        connection = "scm:org-856813@github.com:stripe/stripe-android.git"
+                        developerConnection = "scm:org-856813@github.com:stripe/stripe-android.git"
+                    }
+
+                    licenses {
+                        license {
+                            name = "The MIT License"
+                            url = "https://raw.githubusercontent.com/stripe/stripe-android/master/LICENSE"
+                            distribution = "repo"
+                        }
+                    }
+
+                    developers {
+                        developer {
+                            id = "stripe"
+                            name = "Stripe"
+                        }
+                    }
+                }
             }
             // Creates a Maven publication called "debug".
             debug(MavenPublication) {
```

Comment:
Do we ever want to publish a debug build?

Reply:
We can probably remove the debug publication, but I'll investigate that for a future PR.